src/System.Runtime.Numerics/src/System/Numerics/BigInteger.cs

```diff
@@ -256,21 +256,42 @@ public BigInteger(decimal value)
         {
         }
 
-        public BigInteger(ReadOnlySpan<byte> value)
+        public BigInteger(ReadOnlySpan<byte> value, bool isUnsigned=false, bool isBigEndian=false)
```

Comment:
Have you run any perf tests on the `BigInteger(byte[] value)` ctor?  When adding the `ReadOnlySpan<byte>` ctor, if memory serves I initially had some regressions that I had to work around.  It'd be interesting to understand if these changes regress it at all.